src/pages/BookingsList/useGetBookingsTableColumns.tsx

```diff
@@ -0,0 +1,121 @@
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { IconButton } from '@mui/material';
+import { Download } from '@mui/icons-material';
+import { GridColDef } from '@mui/x-data-grid';
+import { ITicket } from '../../types';
+import { convertTimeStamp } from '../../utils';
+
+interface GridValueGetterParams {
+    row: ITicket;
+}
+
+const useGetBookingsTableColumns = (): GridColDef[] => {
+    const { t } = useTranslation('bookingsList');
+    return [
+        {
+            field: 'pnrNumber',
+            headerName: t('pnrNumber'),
+            renderCell: (params: GridValueGetterParams): JSX.Element => {
+                return (
+                    <>
+                        <Link to="#">{params.row.pnrNumber}</Link>
+                    </>
+                );
+            },
+            headerAlign: 'center',
+        },
+        {
+            field: 'passengerCount',
+            headerName: t('pax'),
+            maxWidth: 50,
+            align: 'right',
+            renderCell: (params: GridValueGetterParams): JSX.Element => {
+                return <p>{params.row.seats.length}</p>;
+            },
+            headerAlign: 'center',
+        },
+        {
+            field: 'tripId',
+            headerName: t('tripId'),
+            maxWidth: 80,
+            align: 'right',
+            renderCell: (params: GridValueGetterParams): JSX.Element => {
+                return <p>{params.row.trip.id}</p>;
+            },
+            headerAlign: 'center',
+        },
+        {
+            field: 'departureDate',
+            headerName: t('date'),
+            maxWidth: 100,
+            renderCell: (params: GridValueGetterParams): JSX.Element => {
+                const { formattedDepartureDate } = convertTimeStamp(
+                    params.row.trip.departureTimestamp
+                );
+                return <p>{formattedDepartureDate}</p>;
+            },
+            headerAlign: 'center',
+        },
+        {
+            field: 'origin',
+            headerName: t('origin'),
+            renderCell: (params: GridValueGetterParams): JSX.Element => {
+                console.log(params);
```

Comment:
Could remove logs to console before merging to develop..

Reply:
Resolved